pkg/common/util/config.go

```diff
@@ -0,0 +1,36 @@
+package util
+
+import (
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/ServiceComb/go-archaius/sources/file-source"
+
+	"github.com/kubeedge/kubeedge/beehive/pkg/common/config"
+)
+
+// LoadConfig is function to Load Configarations
+func LoadConfig() error {
```

Comment:
Why do we create this function? Only test codes need it?

Reply:
yes.

Reply:
yes, this will be common and used for all UT's

Reply:
Is there any way that adding this function without increase the size of kubeedge binary?

Reply:
Done.